src/api/app/controllers/webui/request_controller.rb

```diff
@@ -293,6 +294,10 @@ def conversation
 
   private
 
+  def check_beta_user_redirect
+    redirect_to request_show_path(params[:number], params[:request_action_id]) unless Flipper.enabled?(:request_show_redesign, User.session)
```

Comment:
> Shouldn't we just `raise Pundit::NotAuthorizedError` and let `RescueAuthorizationHandler` handle this?

I wouldn't complicate things for people who are working with requests, I'd prefer to show the request show view (in whatever format they can get according to their beta setup).

If we raise an error they won't know why, they maybe don't even detect that the URL is wrong. Moreover, if they log in because they see the error "Not Authorized", they could be out of beta and see the same error again.

Reply:
Semantically speaking, to the question "is the user authorized to see the info of the request?" the answer is "yes". So they should see it. In one format or another.

Reply:
it's not about authorization. We don't need a logged-in user to see the request page. Previously we rendered show page when the user was not logged in. Now instead of rendering I've just redirected the user

Reply:
okay